homeassistant/components/group.py

```diff
@@ -266,10 +292,20 @@ def icon(self):
         """Return the icon of the group."""
         return self._icon
 
+    @asyncio.coroutine
```

Comment:
You can make this a callback and use `self.hass.loop.create_task(self.async_update_ha_state())` instead of the `yield`